pegasus/sites.v3/code.org/public/dance.haml

```diff
@@ -28,7 +28,7 @@ theme: responsive_full_width
 %section.bg-neutral-light
   .wrapper.centered
     %h2=hoc_s(:dance_featuring_songs_by_short)
-    = view :"dance/featured_artists"
+    = view :"dance/dance_party_featured_artists"
```

Comment:
Renamed this Dance Party view to be more descriptive now that there will be a Music Lab version.